client_test.go

```diff
@@ -180,34 +180,47 @@ func runNewTestClient(ctx context.Context, t *testing.T, config *Config) *Client
 	return client
 }
 
-func Test_Client(t *testing.T) {
+func Test_Client_Standard(t *testing.T) {
```

Comment:
Ugh, I think I was trying to rename this to prevent overzealous test matching when trying to run a single test with `-run`. This is a long issue to explain, so reverting this and will start a separate discussion about it if necessary.